packages/icons/metadata.yml

```diff
@@ -2,13 +2,10 @@ icons:
   - name: add
     friendly_name: Add
     usage: This is a description for usage
-    categories:
-      - Navigation
+    category: Actions
+    subcategory: Navigation
     aliases:
-      - plus
-      - positive
-      - addition
-      - increase
+      - add
```

Comment:
We also seemed to have lost some of the aliases 😮

Reply:
@joshblack this is what happens when i run `scaffold-metadata.js`. it doesn't preserve previous data. we can exclude this file from this PR until it's resolved (or rather until i understand how to resolve!). happy to fix/chat about this!